app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person["name"], person["age"]) for person in people if person
+    ]
+
+    for person in people:
+        if person:
+            person_instance = Person.people[person["name"]]
```

Comment:
The code assumes that the person exists in the `Person.people` dictionary. If the person is not found, this will raise a `KeyError`. Consider adding a check to ensure the person exists in the dictionary before accessing it.